package.json

```diff
@@ -1,19 +1,40 @@
 {
-  "name": "dogs",
+  "name": "@formidablelabs/dogs",
   "version": "1.0.0",
-  "description": "Formidable dogs",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "description": "Formidable Dogs",
+  "author": "@FormidableLabs",
+  "license": "MIT",
   "repository": {
-    "type": "git",
     "url": "git+https://github.com/FormidableLabs/dogs.git"
   },
-  "author": "",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/FormidableLabs/dogs/issues"
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com"
+  },
+  "scripts": {
+    "build": "rimraf dist/ && tsc"
+  },
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "dependencies": {},
+  "devDependencies": {
+    "@commitlint/cli": "^11.0.0",
+    "@commitlint/config-conventional": "^11.0.0",
+    "husky": "^4.3.0",
+    "rimraf": "^3.0.2",
+    "semantic-release": "^17.2.1",
+    "typescript": "^4.0.3"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
```

Comment:
Are there some restrictions for commit messages that we should add to documentation?